libnavigation-ui/src/main/java/com/mapbox/navigation/ui/voice/VoiceInstructionLoader.java

```diff
@@ -121,7 +122,9 @@ void addCachedUrl(String url) {
   }
 
   private void cacheInstruction(String instruction) {
-    requestInstruction(instruction, SSML_TEXT_TYPE, new InstructionCacheCallback(this));
+    if (instruction != null && !TextUtils.isEmpty(instruction)) {
```

Comment:
Same as above re: `instruction != null` is not needed as it's checked in `TextUtils.isEmpty`

Reply:
Same as above